static/app/components/performance/waterfall/treeConnector.tsx

```diff
@@ -14,9 +14,10 @@ const TREE_TOGGLE_CONTAINER_WIDTH = 40;
 export const ConnectorBar = styled('div')<{orphanBranch: boolean}>`
   height: 250%;
 
-  border-left: 1px ${p => (p.orphanBranch ? 'dashed' : 'solid')} ${p => p.theme.border};
+  border-left: 1px ${p => (p.orphanBranch ? 'dashed' : 'solid')} ${p => p.theme.subText};
   top: -5px;
   position: absolute;
+  opacity: 0.4;
```

Comment:
@dashed Only added opacity because `gray300` seems too strong as line color (sadly we don't have a color between `gray200` and `gray300`). An alternative is to use the color package, and set the border color to `Color(theme.gray300).alpha(0.4).string()`. Wdyt?

Reply:
@vuluongj20 Yes, the generated colour string (without the alpha) would be preferred.

Reply:
Let's make sure it doesn't run a huge number of times if we use that computation.

Reply:
Ideally, we'd precompute it and hardcode it, instead of during runtime.

Reply:
@evanpurkhiser @dashed imo, neither `gray200` nor `gray300` works well at full opacity. Here's another solution: we can keep the line color at `gray200`, but increase the line width to 2px. This way the lines are more prominent (thus addressing the original problem) but not too contrasty. Dm'ed Dora and she's cool w/ it. What do y'all think?

<img width="561" alt="Screen Shot 2021-12-09 at 2 41 18 PM" src="https://user-images.githubusercontent.com/44172267/145487659-7af186f7-d1e9-4b09-857f-f52ce5b65016.png">

<img width="561" alt="Screen Shot 2021-12-09 at 2 41 04 PM" src="https://user-images.githubusercontent.com/44172267/145487657-7e6af8b4-7b9d-4342-9fe9-487f2491752a.png">

Reply:
@vuluongj20 Works for me. I'll defer to @getsentry/performance team for any in-depth review with regards to the new changes.

Reply:
@vuluongj20 Since the tree branches have a wider width. Would you be willing to explore making use of an svg for the dashed branches?

Also did you manually set the dashed lines for the new screenshots? If not, that should not occur and is a bug.

Reply:
@dashed Hmm why would we want to use an svg for dashed branches?

And yes, I manually set the dashed lines, to illustrate what it would look like when a connector line has type `orphan`.

Reply:
@vuluongj20 The dashed branches are inconsistent across browsers since they each have different opinions of what `border-style: dashed;`.  In most cases they do not pattern well. 

It's been on my backlog since last year 😄 

I might likely look into it sometime next year.